cumulusci/cumulusci.yml

```diff
@@ -508,6 +508,18 @@ flows:
                 task: update_admin_profile
 
 services:
+  connected_app:
+    description: Configure connected app for connecting to Salesforce orgs
```

Comment:
Consider:
"A connected app is required to connect to and run commands against persistent orgs. See http://cumulusci.readthedocs.io/en/latest/tutorial.html#creating-a-connected-app for more."